tomviz/GenericHDF5Format.cxx

```diff
@@ -450,6 +551,23 @@ bool GenericHDF5Format::read(const std::string& fileName, vtkImageData* image,
     }
   }
 
+  // Look for some common places where there are angles, and
+  // load in the angles if we find them.
+  QVector<double> angles;
+  std::vector<std::string> placesToSearch = { "angle" };
```

Comment:
@cryos Actually, this is the HXN part I think.

Reply:
We should probably add "angles" to the search, I think that will be a common variant.

Reply:
Fixed